src/TitleDetails/ReceivingModal/ReceivingModal.js

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Field } from 'react-final-form';
+import { get } from 'lodash';
+
+import {
+  Button,
+  Col,
+  KeyValue,
+  Modal,
+  ModalFooter,
+  Row,
+  TextArea,
+  TextField,
+} from '@folio/stripes/components';
+import stripesFinalForm from '@folio/stripes/final-form';
+import { FieldSelectFinal } from '@folio/stripes-acq-components';
+
+import { getItemStatusLabel } from '../utils';
+
+const ReceivingModal = ({
+  close,
+  form,
+  handleSubmit,
+  locations,
+  poLineNumber,
+  title,
+}) => {
+  const initialValues = get(form.getState(), 'initialValues', {});
+
+  const footer = (
+    <ModalFooter>
+      <Button
+        buttonStyle="primary"
+        data-test-receive-piece
+        onClick={handleSubmit}
+      >
+        <FormattedMessage id="ui-receiving.piece.actions.receive" />
+      </Button>
+      <Button
+        data-test-receive-piece-cancel
+        onClick={close}
+      >
+        <FormattedMessage id="ui-receiving.piece.actions.cancel" />
+      </Button>
+    </ModalFooter>
+  );
+
+  return (
+    <Modal
+      enforceFocus={false}
```

Comment:
is there a reason to not enforce focus here?

Reply:
removed it